azure-cs-aks-managed-identity/README.md

```diff
@@ -0,0 +1,56 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/blob/master/azure-cs-aks-managed-identity/README.md)
+
+# Azure Kubernetes Service (AKS) Cluster using the native Azure Provider
+
+This example deploys an AKS cluster, creates an Azure User Assigned Managed Identity, and sets credentials to manage access to the cluster.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Configure Azure Credentials](https://www.pulumi.com/docs/intro/cloud-providers/azure/setup/)
+
+### Steps
+
+After cloning this repo, from this working directory, run these commands:
+
+1. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init dev
+    ```
+
+1. Set the Azure region location to use:
+    
+    ```
+    $ pulumi config set azure-native:location westus2
+    ```
+
+1. Stand up the cluster by invoking pulumi
+    ```bash
+    $ pulumi up
+    ```
+
+1. After 3-4 minutes, your cluster will be ready, and the kubeconfig YAML you'll use to connect to the cluster will be available as an output. You can save this kubeconfig to a file like so:
```

Comment:
Link as "...will be available as an [Output](https://www.pulumi.com/docs/concepts/inputs-outputs/#outputs)..."

Reply:
This readme was copied from another example so this could be indicative of a larger, repo wide issue.